test/e2e/specs/wp-gutenboarding-spec.js

```diff
@@ -23,18 +28,46 @@ before( async function () {
 	driver = await driverManager.startBrowser();
 } );
 
-describe( 'Gutenboarding: (' + screenSize + ') @parallel', function () {
+describe( 'Gutenboarding: (' + screenSize + ')', function () {
 	this.timeout( mochaTimeOut );
-	describe( 'Visit Gutenboarding page as a new user', function () {
+	describe( 'Visit Gutenboarding page as a new user @parallel @canary', function () {
+		//const vertical = dataHelper.randomPhrase();
+		const siteTitle = dataHelper.randomPhrase();
+
 		before( async function () {
 			await driverManager.ensureNotLoggedIn( driver );
 		} );
 
 		step( 'Can visit Gutenboarding page and see Onboarding block', async function () {
-			const page = await GutenboardingPage.Visit( driver );
+			const page = await NewPage.Visit( driver );
 			const blockExists = await page.waitForBlock();
 			return assert( blockExists, 'Onboarding block is not rendered' );
 		} );
+
+		step( 'Can see Acquire Intent and set a vertical and site title', async function () {
+			const acquireIntentPage = await AcquireIntentPage.Expect( driver );
+			//sub-step temporary removed
+			//await acquireIntentPage.enterVertical( vertical );
+			await acquireIntentPage.enterSiteTitle( siteTitle );
+			return await acquireIntentPage.goToNextStep();
+		} );
+
+		step( 'Can see Design Selector and select one design', async function () {
+			const designSelectorPage = await DesignSelectorPage.Expect( driver );
+			const designOptionsCount = await designSelectorPage.getDesignOptionsCount();
+			return await designSelectorPage.selectDesign(
+				dataHelper.getRandomInt( 1, designOptionsCount )
```

Comment:
We might wanna make sure to intentionally pick a non-premium design.

Could be either by picking the first one, or ensuring there's no `.design-selector__premium-badge` within the design.